src/Security/RememberLoginHash.php

```diff
@@ -190,14 +204,21 @@ public static function generate(Member $member)
     /**
      * Generates a new hash for this member but keeps the device ID intact
      *
+     * @deprecated 5.3.0 Token renewal will be removed in 6.0.0
      * @return RememberLoginHash
      */
     public function renew()
     {
-        $hash = $this->getNewHash($this->Member());
-        $this->Hash = $hash;
+        // Only regenerate token if configured to do so
```

Comment:
```suggestion
        // Only regenerate token if configured to do so
        Deprecation::notice('5.3.0', 'Will be removed without equivalent functionality');
```
As per https://docs.silverstripe.org/en/5/contributing/release_process/#deprecation